requirements.txt

```diff
@@ -1,2 +1,3 @@
 Flask
 gunicorn
+gitpython
```

Comment:
we are not using gitpython its not working while deployment plus it downloads source each time 
i will change deployment script to download sources during deployment